### ISSUE TITLE
Missing documentation for Enhanced accessibility tagging for PDF exports

### ISSUE THREAD

**Issue by silucrn** [DevExpress/dotnet-eud#213]
It seems that there isn't any documentation currently for https://www.devexpress.com/subscriptions/new-2023-2.xml#reporting-tagged-pdf-export . Is there anyway for this to be added?
